### PR TITLE
Add ability to set custom tags on public and private subnets

### PR DESCRIPTION
**Pull request by TommyM**
Signed-off-by: Tommy Maintz <tommy.maintz@gmail.com>

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -20,12 +20,25 @@ variable "private_subnet_bits" {
...
+  description = "Additional tags to set on the private subnets"
+}
+
 variable "public_subnet_bits" {
   type        = number
   default     = null
   description = "The number of bits used for the subnet mask"
 }
 
+
```
Comment (svanharmelen) [inline variables.tf:35]:
```suggestion
```

File: main.tf [hunk 2]
```diff
@@ -82,7 +82,7 @@ resource "aws_subnet" "private" {
   vpc_id                  = aws_vpc.default.id
 
   tags = merge(
-    var.tags, { "Name" = "${var.stack}-private-${local.az_ids[count.index]}" }
+    var.tags, var.private_subnet_tags, { "Name" = "${var.stack}-private-${local.az_ids[count.index]}" }
```
Comment (svanharmelen) [inline main.tf:85]:
```suggestion
    var.tags,
    var.private_subnet_tags,
    { "Name" = "${var.stack}-private-${local.az_ids[count.index]}" }
```

File: main.tf [hunk 3]
```diff
@@ -70,7 +70,7 @@ resource "aws_subnet" "public" {
   vpc_id                  = aws_vpc.default.id
 
   tags = merge(
-    var.tags, { "Name" = "${var.stack}-public-${local.az_ids[count.index]}" }
+    var.tags, var.public_subnet_tags, { "Name" = "${var.stack}-public-${local.az_ids[count.index]}" }
```
Comment (svanharmelen) [inline main.tf:73]:
```suggestion
    var.tags,
    var.public_subnet_tags,
    { "Name" = "${var.stack}-public-${local.az_ids[count.index]}" }
```

File: variables.tf (hunk 1 above)
Comment (TommyM) [reply variables.tf]: Oops 👯

**Pull request merged**